src/pages/MeetingPage/MeetingPage.tsx

```diff
@@ -62,31 +121,42 @@ const MeetingPage = () => {
       : `${marketingActivitiesProperty.join(',')},${customMeetingAdProperty}`
   }`
 
-  const [updateMemberCreated] = useMutation<hasura.UPDATE_MEMBER_CREATED, hasura.UPDATE_MEMBER_CREATEDVariables>(
-    UPDATE_MEMBER_CREATED,
+  const [updateMemberCreated] = useMutation<hasura.UpdateMemberCreated, hasura.UpdateMemberCreatedVariables>(
+    UpdateMemberCreated,
     { variables: { memberId: currentMemberId! } },
   )
-  const [updateMemberProperties] = useMutation<
-    hasura.UPDATE_MEMBER_PROPERTIES,
-    hasura.UPDATE_MEMBER_PROPERTIESVariables
-  >(UPDATE_MEMBER_PROPERTIES, {
-    variables: {
-      memberPropertiesInput: [
-        {
-          member_id: currentMemberId,
-          property_id: categoryData?.property.find(({ name }) => name === '廣告素材')?.id,
-          value: adPropertyValue,
-        },
-        {
-          member_id: currentMemberId,
-          property_id: categoryData?.property.find(({ name }) => name === '行銷活動')?.id,
-          value: marketingActivitiesPropertyValue,
-        },
-      ],
+  const [updateMemberProperties] = useMutation<hasura.UpdateMemberProperties, hasura.UpdateMemberPropertiesVariables>(
+    UpdateMemberProperties,
+    {
+      variables: {
+        memberPropertiesInput: [
+          {
+            member_id: currentMemberId,
+            property_id: categoryData?.property.find(({ name }) => name === '廣告素材')?.id,
+            value: adPropertyValue,
+          },
+          {
+            member_id: currentMemberId,
+            property_id: categoryData?.property.find(({ name }) => name === '行銷活動')?.id,
+            value: marketingActivitiesPropertyValue,
+          },
+        ],
+      },
+    },
+  )
+
+  const [insertMemberTask] = useMutation<hasura.InsertMemberTask, hasura.InsertMemberTaskVariables>(InsertMemberTask)
+
+  const { data: memberData } = useQuery<hasura.GetMemberByUsername, hasura.GetMemberByUsernameVariables>(
+    GetMemberByUsername,
+    {
+      variables: { appId, username: managerUsername },
     },
-  })
+  )
 
-  if (settings['custom.permission_group.salesLead'] !== '1') {
+  const managerId = managerUsername ? memberData?.member_public[0]?.id || undefined : undefined
+
+  if (settings['custom.permission_group.salesLead'] !== '1' || (managerUsername && !managerId)) {
```

Comment:
I noticed that this condition `settings['custom.permission_group.salesLead'] !== '1'` is commented out in the xuemi repo. Is it necessary here?

Reply:
In the xuemi Repo, the previous people have already comment out, so I ignored it.
It seems like it would affect existing customers if I comment out here, so I left it in.

Reply:
ok